packages/@aws-cdk/aws-route53-patterns/README.md

```diff
@@ -27,4 +49,5 @@ This construct allows creating a simple domainA -> domainB redirect using CloudF
   });
   ```
 
-See the documentation of `@aws-cdk/aws-route53-patterns` for more information.
+The removal policy on the S3 bucket used to store your content is set to `RemovalPolicy.DESTROY`
+When your stack is deleted, the bucket will be physically destroyed.
```

Comment:
Will everything else as well? Any particular reason to call out the bucket here (again, given there isn't any content in it)?

Reply:
I guess it isn't needed to explicitly call it out. I'll remove it

Reply:
the reasoning for including a blurb is because the default for an S3 bucket is to retain on removal. The motivation was to try and be a little transparent about the opinionated setting being used in this pattern.

Reply:
Yeah, I suppose. Counter-argument -- the bucket will have no content, the user gets no reference to it (to add content), and isn't really being used as a bucket so much as a "website redirect mechanism".